src/NuGet.Services.AzureSearch/Analysis/ExactMatchCustomAnalyzer.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.Search.Models;
+
+namespace NuGet.Services.AzureSearch
+{
+    /// <summary>
+    /// Support for case-insensitive exact matching on a field
+    /// in an Azure Search index.
+    /// </summary>
+    public static class ExactMatchCustomAnalyzer
```

Comment:
See [these tests](https://github.com/NuGet/NuGet.Services.Metadata/pull/487/files#diff-e554f728a071933acc5dbd2dc07b949eR9) for tokenization examples